src/network/NetworkInterface.php

```diff
@@ -37,7 +37,7 @@ interface NetworkInterface{
 	 */
 	public function start() : void;
 
-	public function setName(string $name) : void;
+	public function setName(string $name, string $subName = "PocketMine-MP") : void;
```

Comment:
This is BC-breaking.

It would also make more sense to make this nullable and have null as the default, so that the callee can decide what to do if nothing was specified (e.g. to use previous value, or to use Server->getSubMotd())

Reply:
Further, `PocketMine-MP` should not be hardcoded all over the place.

Reply:
Ok, i will change this to nullable.